test/specs/components/Icon/Icon-test.tsx

```diff
@@ -33,7 +33,7 @@ describe('Icon', () => {
         const renderedComponent = mountWithProviderAndGetComponent(
           Icon,
           <Icon name="fontIcon" />,
-          null,
+          undefined,
```

Comment:
This arguments is `options`, `null` and `undefined` aren't equal for default args:

```js
function getValue(value = { foo: 'bar' }) {
  return value
}

console.log(getValue(null)) // => null
console.log(getValue(undefined)) // {foo: "bar"}
```

Reply:
great catch!